bool use_defaults
float32 x
float32 y
float32 depth
bool radial
bool delete_prev_traj
---
bool success
string message
